### PR TITLE
Bumped yarn versions and minor optimizations

### PR DESCRIPTION
**Pull request by ctran4347**

### CODE REVIEW

File: .github/workflows/ensure-prettier.sh [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-# This is used by the GitHub Actions 'lint-and-format' job.
-
-set -euo pipefail
-
-yarn prettier
-
-if git status --porcelain | grep .; then
-    echo "Changes detected; did you format your code with 'yarn prettier'?"
-    exit 1;
-fi;
```
Comment (awpala) [inline .github/workflows/ensure-prettier.sh:14]: I think we should keep linting & formatting in there to keep our code formatting consistent in the repo

File: .storybook/main.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import type { StorybookConfig } from '@storybook/nextjs'
+const config: StorybookConfig = {
+	stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```
[the remark this answers is not in the document]
Comment (ctran4347) [reply .storybook/main.ts:3]: This was auto-generated by storybook

[the remark this answers is not in the document]
Comment (omshub-admin) [reply .storybook/main.ts:3]: Eh that's fine, let's just leave it for now, not a hill I'm gonna die on at this point 😬

[the remark this answers is not in the document]
Comment (awpala) [reply .storybook/main.ts:3]: > This was auto-generated by storybook

Eh that's fine, let's just leave it for now, not a hill I'm gonna die on at this point 😬

**Review by awpala — APPROVED** [review]

**Pull request merged**